webapp/channels/src/components/integrations/abstract_oauth_app.test.tsx

```diff
@@ -85,12 +90,12 @@ describe('components/integrations/AbstractOAuthApp', () => {
         const wrapper = shallow(
             <AbstractOAuthApp {...props}/>,
         );
-
-        wrapper.instance().updateName({target: {value: 'new name'}});
+        const evt = {preventDefault: jest.fn(), target: {value: 'new name'}} as any;
+        (wrapper.instance() as AbstractOAuthApp).updateName(evt);
```

Comment:
If you can give the type to shallow function you dont need to type casting. Example would be 
```ts
const wrapper = shallow<AbstractOAuthApp>(
           <AbstractOAuthApp {...props}/>,
        );
```

Reply:
didn't know this, Thanks